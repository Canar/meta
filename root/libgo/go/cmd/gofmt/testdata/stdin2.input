../../../../../../gcc/libgo/go/cmd/gofmt/testdata/stdin2.input