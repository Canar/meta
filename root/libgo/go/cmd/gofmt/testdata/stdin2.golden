../../../../../../gcc/libgo/go/cmd/gofmt/testdata/stdin2.golden